src/main/app/claims/models/countyCourtJudgment.ts

```diff
@@ -10,7 +10,7 @@ export class CountyCourtJudgment {
                public paymentOption?: string,
                public paidAmount?: number,
                public repaymentPlan?: RepaymentPlan,
-               public payBySetDate?: Moment,
```

Comment:
this isn't right, we should be using a moment so we can do date work on it...

Reply:
👍